modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task.jsp

```diff
@@ -69,9 +69,11 @@ renderResponse.setTitle(workflowTaskDisplayContext.getHeaderTitle(workflowTask))
 				request.removeAttribute(WebKeys.SEARCH_CONTAINER_RESULT_ROW);
 				%>
 
-				<liferay-util:include page="/workflow_task_action.jsp" servletContext="<%= application %>">
-					<liferay-util:param name="mvcPath" value="/edit_workflow_task.jsp" />
-				</liferay-util:include>
+				<c:if test="<%= workflowTaskDisplayContext.isWorkflowTaskReadOnly() == false %>">
```

Comment:
@rijema we can negate this to simplify:

`!workflowTaskDisplayContext.isReadOnly()`

please replace all `workflowTaskDisplayContext.isWorkflowTaskReadOnly() == false` with `!workflowTaskDisplayContext.isReadOnly()`

Thanks!

Reply:
Done!